pendulum/datetime.py

```diff
@@ -41,7 +41,7 @@
 from pendulum.utils._compat import PY38
 
 if TYPE_CHECKING:
-    from typing import Literal
+    from pendulum.utils._compat import Literal
```

Comment:
This should be changed to `from typing_extensions import Literal` so that pyright is supported